src/components/auth/login/Login.jsx

```diff
@@ -46,29 +52,37 @@ const LogInForm = () => {
       <Form.Control
         type="password"
         placeholder="Password"
-        ref={passwordValue}
+        onChange={(e) => setPasswordValue(e.target.value)}
       />
-      <Button
-        variant="info"
-        size="md"
-        className="loginButton"
-        onClick={`${isLoggedIn ? userLogout : userLogin}`}
-      >
-        {`${isLoggedIn ? "LOG OUT" : "LOG IN"}`}
-      </Button>
     </Form.Group>
   );
 
   return (
-    <>
+    <Row>
       {isAdmin && <AddProducts />}
       <Col className="signupForm">
-        <Form>
+        <Form
+          onSubmit={(e) => {
+            if (isLoggedIn === true) {
+              userLogout(e);
+            } else {
+              userLogin(e);
+            }
+          }}
+        >
           {emailGroup}
           {passwordGroup}
+          <Button
+            variant="info"
+            size="md"
+            className="loginButton"
+            type="submit"
+          >
+            {`${isLoggedIn ? "LOG OUT" : "LOG IN"}`}
```

Comment:
Log out, Log in, and use text-transform to make it capitalized (more maintainable)